### DISCUSSION TITLE
How to get the output directory on Linux?

### DISCUSSION THREAD

**Discussion by frodik13** [unoplatform/uno#15568] [Q&A] [answered]
I want to get the directory where the files are collected during the build. This is the directory I need - ..bin\x64\Debug\net8.0-windows10.0.19041.0\win-x64\AppX. I get it fine for Windows, but I just can’t figure out how to get it in Linux. I essentially need to get this ..\bin\Debug\net8.0. Help me please.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. the `.Windows` head cannot be built entirely on Linux, mostly because the WinAppSDK tooling is windows specific. It's the same for iOS apps.
